units/systemd-networkd-persistent-storage.service

```diff
@@ -0,0 +1,24 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Notify if the persistent storage for systemd-networkd.service is ready
+Documentation=man:networkctl(1)
+ConditionCapability=CAP_NET_ADMIN
+After=systemd-networkd.service
+BindsTo=systemd-networkd.service
+ConditionPathExists=!/etc/initrd-release
+
```

Comment:
hmm, i think this should run during early boot already (i.e. have DefaultDependencies=no), but order itself after systemd-remount-fs.service and RequiresMountsFor=/var/lib/systemd/network, so that it is not called before that dir is there and writable.